tests/testSubCommands.nim

```diff
@@ -34,6 +36,10 @@ elif clone:
   echo "clone subcommand chosen"
   echo "cloning ", gitUrl, "..."
 
+elif push:
+  echo "push subcommand chosen"
+  echo "pushin ..."
+
 else:
   echo "no subcommands have been chosen"
 
```

Comment:
To fix the tests, first modify the `if testing` conditional to:

```nim
if testing:
  if add:
    doAssert(filenames == @["foo", "bar", "baz"])
    doAssert(force == true)
    doAssert(interactive == false)
    doAssert(clone == false)
  else:
    doAssert(push == true)
else:
  doAssert(false)
```

then change the tests.json file as described below. That should do it ;) .

Reply:
Tests now pass!